src/main/entity/Restaurant.ts

```diff
@@ -33,12 +33,18 @@ export class Restaurant {
   /**
    * Latitude coordinates of restaurant
    * */
-  @Column('real')
-  lat: number;
+  @Column({
+    type: 'real',
+    nullable: true,
+  })
+  lat: number | null;
```

Comment:
I think this can just be
```suggestion
  lat?: number;
```

Then the constructor doesn't need the `| null` part